zabbix_metrics_watcher.py

```diff
@@ -147,8 +152,8 @@ def _send_zabbix_message(zabbix_host, osbs_master, key, value, print_command=Tru
 
 def filter_completed_builds(completed_builds):
     # Remove all completed_builds which are not within this hour
-    now = int(time())
-    return {k: v for k, v in completed_builds.items() if (v - now) < 3600}
+    now = int(timegm(gmtime()))
```

Comment:
Reading the documentation, I have no clue why `time.time()` would not be equivalent to `calendar.timegm(time.gmtime())`. Is that really correct?

We want `now` to be the number of seconds since the Epoch, don't we?